pkg/actors/actors.go

```diff
@@ -437,96 +437,115 @@ func (a *actorsRuntime) constructActorStateKey(actorType, actorID, key string) s
 }
 
 func (a *actorsRuntime) connectToPlacementService(placementAddress, hostAddress string, heartbeatInterval time.Duration) {
-	log.Infof("starting connection attempt to placement service at %s", placementAddress)
-	stream := a.getPlacementClientPersistently(placementAddress, hostAddress)
-	log.Infof("established connection to placement service at %s", placementAddress)
+	// isConnAlive represents the status of stream channel. This must be changed in receiver loop.
+	// This flag reduces the unnecessary request retry.
+	var isConnAlive bool = true
+	stream := a.newPlacementStreamConn(placementAddress)
 
+	// Establish receive channel to retrieve placement table update
 	go func() {
 		for {
-			host := placementv1pb.Host{
-				Name:     hostAddress,
-				Load:     1,
-				Entities: a.config.HostedActorTypes,
-				Port:     int64(a.config.Port),
-				Id:       a.config.AppID,
+			// Wait until stream is reconnected.
+			if !isConnAlive || stream == nil {
+				time.Sleep(placementReconnectInterval)
```

Comment:
For the sake of not browning out an overloaded placement service, can we do exponential backoff on our retry interval?

Reply:
for this case, linear retry is enough because this code inside this loop does not retry to establish the connection. 

We may need to add exponential backoff in `newPlacementStreamConn`

As I describe in the comment, this pr focuses on the current reported issue. Once this PR is merged, I will refactor to make it testable and add more tests.